### PR TITLE
Bugfix - Skip Dracut Module and Mariner.cfg Update with no Verity Cfg.

### PR DESCRIPTION
**Pull request by liulanze**
Bugfix - Skip Dracut Module and Mariner.cfg Update with no Verity Cfg.

###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [ ] The toolchain has been rebuilt successfully (or no changes were made to it)
- [ ] The toolchain/worker package manifests are up-to-date
- [ ] Any updated packages successfully build (or no packages were changed)
- [ ] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [ ] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [ ] All package sources are available
- [ ] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [ ] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [ ] All source files have up-to-date hashes in the `*.signatures.json` files
- [ ] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [ ] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Title says, skip Dracut module re-building and Mariner.cfg updating when users do NOT customize Verity related configs.

###### Change Log  

- Add an additional if statement checking for Verity field exists or not.

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Local

### CODE REVIEW

File: toolkit/tools/pkg/imagecustomizerlib/customizeutils.go
Comment (vinceaperri) [inline toolkit/tools/pkg/imagecustomizerlib/customizeutils.go:1]: May be better to do the check outside the function call so you don’t have to add the argument.

**Review by vinceaperri — APPROVED** [review]

**Review by romoh — APPROVED** [review]

File: toolkit/tools/pkg/imagecustomizerlib/customizeutils.go
Comment (liulanze) [reply toolkit/tools/pkg/imagecustomizerlib/customizeutils.go:1]: I see, I saw these functions do the special case checking at the very beginning, so I follow the style. Like for example function `updateHostname` or `runScripts`. I will leave it as is right now. Thx.

**Comment by amritakohli** [comment]
Can you please update / clean up the PR description?

File: toolkit/tools/pkg/imagecustomizerlib/customizeutils.go
Comment (mfrw) [inline toolkit/tools/pkg/imagecustomizerlib/customizeutils.go:102]: +1 on doing the check before the function call to avoid adding an argument.

**Comment by ameyapu** [comment]
Thanks for this change.  LGTM!  Once the merge happens, please also update the corresponding artifact.

**Review by jiria — APPROVED** [review]

**Review by anphel31 — APPROVED** [review]
Please fill out the PR description, and in particular ensure the "sudo make go-tidy-all and sudo make go-test-coverage" tests pass

**Pull request merged**
